types/tests/chart_types.ts

```diff
@@ -0,0 +1,42 @@
+import { Chart } from '../index.esm';
+
+const chart = new Chart('chart', {
+  type: 'bar',
+  data: {
+    labels: ['1', '2', '3'],
+    datasets: [{
+      data: [1, 2, 3]
+    },
+    {
+      data: [1, 2, 3]
+    }],
+  }
+});
+
+const chart2 = new Chart('chart', {
+  type: 'bar',
+  data: {
+    labels: ['1', '2', '3'],
+    datasets: [{
+      type: 'line',
+      data: [1, 2, 3]
```

Comment:
could add some type specific options to make sure it works as expected?

Reply:
Guess I have all the main use cases on which it might have been able to break covered with testing right now